src/routes/(inner)/examples/popovers/Example.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
```

Comment:
The popover appears to be a bit buggy.

- [ ] First, it's on-by-default, resulting in the position shift on page load. Try refreshing the page a few times to preview this.
- [ ] Second, when the popup is closed, it doesn't react to new clicks to open it.